docs/about/stability.rst

```diff
@@ -109,20 +154,29 @@ PlasmaPy's planned and existing subpackages are:
                 <span class="dev"></span>
             </td>
             <td>
-                The plan for PlasmaPy's base classes is being planned in
-                PLEP 7, which is in the process of being written.  The
-                existing functionality is unstable.
+                This subpackage is currently under development.
             </td>
         </tr>
         <tr>
             <td>
-                plasmapy.formulary
+                plasmapy.simulation
+            </td>
+            <td align='center'>
+                <span class="dev"></span>
+            </td>
+            <td>
+                This subpackage is currently under development.
```

Comment:
This is pretty boilerplate text, but we haven't really been updating the stability matrix, so I wanted to put something in that won't go out of date for a while.